### DISCUSSION TITLE
Problemi accesso da remoto

### DISCUSSION THREAD

**Discussion by SimoP79** [indomus/forum#2682] [HUB: Home Assistant]
Buongiorno a tutti, ho installato HASSIO su un raspberry 3B+ seguendo tutte le guide su indomus ma ho un problema di accesso da remoto: ho seguito tutta la procedura descritta per l'utilizzo con duckdns; ho anche provveduto ad aprire la porta 8123 sul router seguendo una guida di tp-link ma niente da fare, eseguendo i test sulla porta risulta comunque chiusa e quindi non riesco ad avere accesso tramite l'app. Ho anche fatto una prova disabilidando windows firewall ma non è cambiato nulla!
Suggerimenti su come posso risolvere?

**Comment by snakuzzo** [comment]
Ciao.
Mi spieghi cosa c'entra il firewall di Windows se hai installato HA OS su un RPI ? Te lo chiedo perché a pelle mi verrebbe da dire che stai facendo le cose totalmente a caso senza alcun senso logico.
Il fatto che tu abbia seguito la procedura, non vuol dire che tu l'abbia fatto correttamente...quindi ti consiglierei di riguardare passo per passo tutto quello che hai fatto.
In merito al discorso port forwarding, devi sapere tu se l'hai configurato correttamente...non possiamo dirtelo noi.

Assicurati che al tuo RPI sia assegnato sempre lo stesso IP (questa cosa si configura sul router...cerca come fare) e poi configuri il port forwarding per mappare la porta TCP esterna (immagino tu voglia utilizzare sempre la 8123) verso la TCP 8123 dell'IP LAN assegnato al RPI.

**Reply by SimoP79** [reply]
scusa ho sbagliato a scrivere! windows firewall non c'entra nulla! ho provato disabilitando il firewall del router.
L'ip interno assegnato all'RPI è sempre lo stesso e per il port fowarding ho seguito la procedura descritta da tp-link, ma verificando su portchecker mi risulta che la porta e chiusa, quindi immagino che il problema sia di impostazione sul router; se per caso qualcuno conosce i tp-link e può darmi qualche dritta ne sarei felice (ho un router 4G MR200).

**Reply by snakuzzo** [reply]
Ma guarda che il firewall non devi disabilitarlo

**Reply by Maspar79** [reply]
Sbaglio ho il router in questione sfrutta la rete mobile? Se così fosse la tua sim potrebbe non avere un IP pubblico.

**Reply by SimoP79** [reply]
Esatto, è un router che sfrutta una sim! La mia zona non è ancora coperta dalla fibra e per il momento ho in funzione questo router con una sim fornita da Alpsim (connessione dati 4G senza limiti di traffico).
Se fosse la sim a non avere un ip pubblico il problema è risolvibile?

**Reply by Maspar79** [reply]
Su indomus è stato spiegato come fare questo è il link: https://indomus.it/guide/collegarsi-da-remoto-a-home-assistant-via-reti-provider-tradizionali-o-che-usino-nat-via-cloudflare/

**Reply by SimoP79** [reply]
Grazie! non avevo trovato questa guida!
Ho seguito passo passo tutta la procedura descritta, al termine aggiornando il registro appare turro come descritto ma se immetto il dominio da me attivato l'app non si connette comunque!

**Reply by snakuzzo** [reply]
@SimoP79 perdonami...ma secondo te uno come può dare una mano in base alla frase "l'app non si connette comunque" ?
 Sfortunatamente non abbiamo la sfera di cristallo e se qualcuno riesce a darti una risposta al volo risolvendo, può farlo solo per una grandissima botta di cu...o :)
Devi dettagliare per bene cosa hai fatto e che tipo di problema hai.
